app/Enum/CacheKey.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Enum;
+
+class CacheKey
+{
+    public const LeaderboardFriends        = "LeaderboardFriends";
+    public const LeaderboardGlobalPoints   = "LeaderboardGlobalPoints";
+    public const LeaderboardGlobalDistance = "LeaderboardGlobalDistance";
+    public const LeaderboardMonth          = "LeaderboardMonth";
+
+    public static function getFriendsLeaderboardKey(int $user_id): string {
+        return self::LeaderboardFriends + '-for-' + $user_id;
```

Comment:
Did you test your code? :D

```suggestion
        return self::LeaderboardFriends . '-for-' . $user_id;
```